File: Modulo2-ObjectiveC-Lorena/Exercicio5.m
//
//  main.m
//  Exercicio5-C
//
//  Created by Lorena Rodrigues Bruno on 05/09/2018.
//  Copyright © 2018 Lorena Rodrigues Bruno. All rights reserved.
//

#import <Foundation/Foundation.h>

int main(int argc, const char * argv[]) {
    @autoreleasepool {
        int mes;
       
        do{
            printf("Digite um número de 1 a 12: \n*Para sair, digite 0\n");
            scanf("%i", &mes);
            switch (mes) {
            case 1:
                NSLog(@"Janeiro");
                break;
            case 2:
                NSLog(@"Fevereiro");
                break;
            case 3:
                NSLog(@"Março");
                break;
            case 4:
                NSLog(@"Abril");
                break;
            case 5:
                NSLog(@"Maio");
                break;
            case 6:
                NSLog(@"Junho");
                break;
            case 7:
                NSLog(@"Julho");
                break;
            case 8:
                NSLog(@"Agosto");
                break;
            case 9:
                NSLog(@"Setembro");
                break;
            case 10:
                NSLog(@"Outubro");
                break;
            case 11:
                NSLog(@"Novembro");
                break;
            case 12:
                NSLog(@"Dezembro");
                break;
            default:
                NSLog(@"Mês inválido");
                break;
            }
            
        } while (mes != 0);
    }
    
    return 0;
}

File: Modulo2-Lorena/exercicio5.swift
//
//  main.swift
//  Exercicio5
//
//  Created by Lorena Rodrigues Bruno on 23/08/2018.
//  Copyright © 2018 Lorena Rodrigues Bruno. All rights reserved.
//

import Foundation

var mes : String! = String()

repeat{
    print("Digite um número de 1 a 12: \n*Para sair, digite 0")
    mes = readLine()
    switch mes {
    case "1":
        print("Janeiro\n")
    case "2":
        print("Fevereiro\n")
    case "3":
        print("Março\n")
    case "4":
        print("Abril\n")
    case "5":
        print("Maio\n")
    case "6":
        print("Junho\n")
    case "7":
        print("Julho\n")
    case "8":
        print("Agosto\n")
    case "9":
        print("Setembro\n")
    case "10":
        print("Outubro\n")
    case "11":
        print("Novembro\n")
    case "12":
        print("Dezembro\n")
    default:
        print("Mês inválido\n")
    }
    
} while mes != "0"
